packages/cli/src/cli.ts

```diff
@@ -318,18 +351,22 @@ VAR2=Value2`);
 }
 
 readConf(confFile)
+    .then(overrideConfig)
     .then((conf) => {
         return new DefaultServient(clientOnly, conf);
     })
     .catch((err) => {
         if (err.code === "ENOENT" && !confFile) {
             console.warn("[cli]", `WoT-Servient using defaults as '${defaultFile}' does not exist`);
-            return new DefaultServient(clientOnly);
         } else {
             console.error("[cli]", "WoT-Servient config file error:", err.message);
             process.exit(err.errno);
         }
     })
+    .then(overrideConfig)
```

Comment:
you're calling overrideConfig twice, why?

Reply:
Because the promise chain is made as follows:
- read config file
- then, use the config to start servient
- catch, if any error occurred

Problem is that with this flow it catch an exception whenever I give no config file path.
So I modified the chain like that:
- read config file
- then, assuming the file reading gave no I/O errors, override the config with parameters passed via CLI
- catch, if any error occurred
- then, try to override the config again. Because, if the above catch was called, it means we still have no valid config set at this time
- then, start the servient with the resulting configuration

Reply:
Can't you just do:
`reaFile -> catch -> override -> new Default`

Reply:
You are right, but now catch's `process.exit(err.errno)` should no longer exist, isn't it?
Because If there's any error with the config we override it / throw an error in the second catch branch